### PR TITLE
Get All/Resolve All Feature for Retrieve Action, issue #76

### PR DESCRIPTION
**Pull request by mwittig**
See issue #76

### CODE REVIEW

**Comment by mwittig** [comment]
Sorry, I wasn't aware of the required DCO claim. I have updated the commit message accordingly.

**Comment by caroline-church** [comment]
@mwittig I think you might still have DCO sign off problems

**Comment by mwittig** [comment]
I have changed the sign-off messages to make DCO robot happy. In my opinion the robot check is too rigid as it does a case sensitive (!) match for the e-mail address. Moreover it should not check the user name as it maps to my local `git config user.name`. I can change it at any point in time as it not part of the Github user profile.

File: packages/node-red-contrib-composer/nodes/hyperledger-composer.js [hunk 1]
```diff
@@ -213,7 +223,12 @@ module.exports = function (RED) {
                     return businessNetworkConnection.getParticipantRegistry(modelName)
                         .then((participantRegistry) => {
                             node.log('got participant registry');
-                            return participantRegistry.get(id);
+                            if (id == null) {
+                                return assetRegistry.getAll();
```
Comment (caroline-church) [inline packages/node-red-contrib-composer/nodes/hyperledger-composer.js]: shouldn't this be partcipantRegistry?

Comment (caroline-church) [inline packages/node-red-contrib-composer/nodes/hyperledger-composer.js]: You can also do a resolveAll() on a participant registry too

Comment (mwittig) [reply packages/node-red-contrib-composer/nodes/hyperledger-composer.js]: Agree. It's a stupid copy-paste error. I'll fix that as part of the next commit

Comment (mwittig) [reply packages/node-red-contrib-composer/nodes/hyperledger-composer.js]: OK. The current release does not handle resolve case for participant retrieval and I thought in the first place it is perhaps not required for participant objects.

Comment (caroline-church) [reply packages/node-red-contrib-composer/nodes/hyperledger-composer.js]: I think that's probably a bug so if you could add it as part of your implementation that would be really good

**Comment by mwittig** [comment]
Please give some more time to revise the README and the help node texts.

**Comment by caroline-church** [comment]
@mwittig any update on these changes?

**Comment by mwittig** [comment]
@caroline-church Sorry for the delay at my end. I promise to finish up asap. As I am travelling this week I am shooting for early next week.

**Comment by caroline-church** [comment]
@mwittig It has been  month since your last update, are you intending to update your pull request?

**Comment by mwittig** [comment]
Sorry for the delay. I think I have covered all your review comments. Regarding your request to "add some info to the README.md to show how to use this feature" I have decided to add some text to the info for the `Hyperledger-Composer-Mid` node instead. I think the README pretty generic and is better to have the specific information available as part of the editor info view. Hope, this ok.

**Pull request merged**